src/components/Votes.jsx

```diff
@@ -0,0 +1,43 @@
+import { useState } from "react";
+import { patchArticle } from "../../Utils/api";
+
+const Votes = ({ newArticle }) => {
+  const originalVote = newArticle.votes;
+  const [newVotes, setNewVotes] = useState(originalVote);
+  const [voteChange, setVoteChange] = useState(0);
+  const [err, setErr] = useState(null);
+
+  const updateVotes = (changeValue) => {
+    if (voteChange === 0) {
+      patchArticle(newArticle.article_id, changeValue);
```

Comment:
As it was mentioned in previous comments, when something goes wrong and there's an error when patching the votes, an error message should be displayed and the number of votes should be reverted to the amount before the click. You can set this up in a catch() block.